docker/package/scripts/tezos-accuser-start

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/bash
+
+# SPDX-FileCopyrightText: 2020 TQ Tezos <https://tqtezos.com/>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+set -euo pipefail
+
+# $PROTOCOL should be defined in the system unit environment
+accuser="/usr/bin/tezos-accuser-$PROTOCOL"
+
+accuser_dir="$DATA_DIR"
+
+accuser_config="$accuser_dir/config"
+mkdir -p "$accuser_dir"
+
+if [ ! -f "$accuser_config" ]; then
+    "$accuser" --base-dir "$accuser_dir" \
+               --endpoint "$NODE_ENDPOINT" \
+               config init --output "$accuser_config" >/dev/null 2>&1
+else
+    "$accuser" --base-dir "$accuser_dir" \
+               --endpoint "$NODE_ENDPOINT" \
+               config update >/dev/null 2>&1
+fi
+
+exec "$accuser" --base-dir "$accuser_dir" \
```

Comment:
Why is `exec` needed?

Reply:
thus, `tezos-accuser` will exchange bash process